Podverse/SyncPodcast.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  SyncPodcast.swift
+//  Podverse
+//
+//  Created by Mitchell Downey on 1/20/18.
+//  Copyright © 2018 Podverse LLC. All rights reserved.
+//
+
+import Foundation
+
+import Foundation
+import UIKit
+
+class SyncPodcast {
```

Comment:
This class name confused me before too. It is a Verb instead of a Noun. Maybe renamed it to SyncedPodcast or SyncablePodcast or something like that? (Tiny lesson from my SE 401 class: Verbs=Functions, Nouns=Classes, Adjectives=Properties)